tests/dialects/test_oracle.py

```diff
@@ -6,6 +6,7 @@ class TestOracle(Validator):
     dialect = "oracle"
 
     def test_oracle(self):
+        self.validate_identity("SELECT JSON_OBJECT(k1: v1 FORMAT JSON, k2: v2 FORMAT JSON)")
```

Comment:
This is the use case (same syntax applies to `JSON_ARRAY`) that triggered the need to refactor the parsing of `FORMAT JSON` - see Oracle's docs for reference.